src/common/BaseManager.ts

```diff
@@ -24,8 +26,14 @@ export default class BaseManager implements Manager {
 
     reactEl: HTMLElement;
 
-    constructor({ location = 1, referenceEl }: Options) {
+    resinTags: Record<string, string>;
+
+    constructor({ location = 1, referenceEl, resinTags = {} }: Options) {
         this.location = location;
+        this.resinTags = {
```

Comment:
Can this be passed to `insert` as an argument instead of stored as an instance property?

Reply:
Good call